Doc/library/intro.rst

```diff
@@ -64,3 +64,57 @@ Notes on availability
   libc version, then both conditions must hold. For example a feature with note
   *Availability: Linux >= 3.17 with glibc >= 2.27* requires both Linux 3.17 or
   newer and glibc 2.27 or newer.
+
+.. _wasm-availability:
+
+Notes on WebAssembly platforms
+==============================
```

Comment:
```suggestion
WebAssembly platforms
---------------------
```

I'd suggest making this a subsection of the top-level "Notes on availability" section, since this is a subtopic of it (availability -> WASM availability), and it also makes it a part of the section users reach when they click the link in the ``availability`` directive, which is now heavily being used for WASM (since it links to the "Availability" section) rather than a separate parallel section. It also reduces redundancy in the title.